src/parser.c

```diff
@@ -1324,111 +1318,112 @@ bool flightLogParse(flightLog_t *log, int logIndex, FlightLogMetadataReady onMet
-                            log->stats.frame[frameType->marker].bytes += lastFrameSize;
-                            log->stats.frame[frameType->marker].sizeCount[lastFrameSize]++;
-                            log->stats.frame[frameType->marker].validCount++;
-                        } else {
-                            log->stats.frame[frameType->marker].desyncCount++;
-                        }
+                    if (frameAccepted) {
+                        //Update statistics for this frame type
+                        log->stats.frame[frameType->marker].bytes += frameSize;
+                        log->stats.frame[frameType->marker].sizeCount[frameSize]++;
+                        log->stats.frame[frameType->marker].validCount++;
                     } else {
-                        //The previous frame was corrupt
-
-                        //We need to resynchronise before we can deliver another main frame:
-                        private->mainStreamIsValid = false;
-                        log->stats.frame[frameType->marker].corruptCount++;
-                        log->stats.totalCorruptFrames++;
-
-                        //Let the caller know there was a corrupt frame (don't give them a pointer to the frame data because it is totally worthless)
-                        if (onFrameReady)
-                            onFrameReady(log, false, 0, frameType->marker, 0, frameStart - private->stream->data, lastFrameSize);
-
-                        /*
-                         * Start the search for a frame beginning after the first byte of the previous corrupt frame.
-                         * This way we can find the start of the next frame after the corrupt frame if the corrupt frame
-                         * was truncated.
-                         */
-                        private->stream->pos = frameStart + 1;
-                        frameType = NULL;
-                        prematureEof = false;
-                        private->stream->eof = false;
-                        continue;
+                        log->stats.frame[frameType->marker].desyncCount++;
+                    }
+                } else {
+                    //The previous frame was corrupt
+
+                    //We need to resynchronise before we can deliver another main frame:
+                    private->mainStreamIsValid = false;
+                    log->stats.frame[frameType->marker].corruptCount++;
+                    log->stats.totalCorruptFrames++;
+
+                    //Let the caller know there was a corrupt frame (don't give them a pointer to the frame data because it is totally worthless)
+                    if (onFrameReady) {
+                        onFrameReady(log, false, 0, frameType->marker, 0, frameStart - private->stream->data, frameSize);
                     }
+
+                    /*
+                        * Start the search for a frame beginning after the first byte of the previous corrupt frame.
```

Comment:
Indentation.

This type of comments is meant to be arranged so that all the asterisks at the beginning form a vertical line, like so:

```
/*
 * This is a comment
 */
```